java-hello-world-maven/README-Cloud-Shell.md

```diff
@@ -48,14 +46,31 @@ GraalVM Enterprise Native Image and JDK 17 are preinstalled in Cloud Shell.
     The current managed java version is set to graalvmeejdk-17.0.4.
     ```
 
-## Step 3: [OPTIONAL] Confirm Software Version and Environment Variables
+## Step 3: [OPTIONAL] Confirm software version and environment variables
+
+This step is optional - [Check software version and environment variables](../_common/README-check-version-env-vars.md)
+
+
+## Step 4: Setup project, build and run as a JAR
+
+1. Git clone this sample folder.
+
+    ```shell
+    git init graalvmee-java-hello-world-maven
```

Comment:
Why "init" and not "clone"?

Reply:
The group of commands together clones just a single sample folder, not the entire graalvm-demos repo.